pages/common/peerflix.md

```diff
@@ -0,0 +1,19 @@
+# peerflix
+
+> Stream torrents to your favorite player.
+
+- Stream a torrent:
+
+`peerflix "{{torrent url|magnet link}}"`
+
+- Stream a torrent to a specific player:
+
+`peerflix "{{torrent url|magnet link}}" --{{player}}`
+
+- Stream a torrent to VLC:
```

Comment:
Is this needed considering the above example?

Reply:
probably not